src/stream.rs

```diff
@@ -133,6 +133,7 @@ async fn handle_event(state: &mut State, event: Event) -> Result<(), Error> {
                 Some(Initiator {
                     initiator: Some(initiator::Initiator::Unit(unit)),
                 }),
+            ..
```

Comment:
Is this supposed to be here?

Reply:
Yes, the struct has now another property, something the compiler would complain about. This is the syntax for telling the compiler that we don't care about any other property of the struct 👍

Reply:
Nice, Today I learned.